### DISCUSSION TITLE
Can someone recommend a more helpful forum to start learning Git Hub

### DISCUSSION THREAD

**Discussion by JoahfromAlberta** [community/community#72994] [New to GitHub]
### Select Topic Area

Product Feedback

### Body

Can anyone recommend another forum (other than Reddit) that can help learn Get Hub  People might be too busy to help on this forum.

**Comment by ktauchathuranga** [comment]
i think the best solutions is to learn github is, watching youtube videos,
great videos out there,  i will recommend you some,
[Git and GitHub for Beginners - Crash Course](https://www.youtube.com/watch?v=RGOj5yH7evk&pp=ygUUZ2l0aHViIGZ1bGwgdHV0b3JpYWw%3D)
[Complete Git and GitHub Tutorial](https://www.youtube.com/watch?v=apGV9Kg7ics&pp=ygUUZ2l0aHViIGZ1bGwgdHV0b3JpYWw%3D)
[Git & GitHub Crash Course 2023](https://www.youtube.com/watch?v=ulQA5tjJark&pp=ygUUZ2l0aHViIGZ1bGwgdHV0b3JpYWw%3D)

(you should use forums, after watching these videos and when you are stuck somewhere, and before posting take a look to see if anyone answered that question before)
happy coding 🖤

**Reply by JoahfromAlberta** [reply]
Thank you for your reply, it is my honour to use Git Hub.

**Comment by queenofcorgis** [comment]
Welcome to GitHub and the GitHub Community @JoahfromAlberta  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!

**Reply by JoahfromAlberta** [reply]
Thank you for the warm welcome, it is a steep learning curve that equates the height of advancements you have established.
